plenum/common/util.py

```diff
@@ -346,7 +346,7 @@ def z85_to_friendly(z):
 
 def runWithLoop(loop, callback, *args, **kwargs):
     if loop.is_running():
-        loop.call_soon(asyncio.async, callback(*args, **kwargs))
+        loop.call_soon(asyncio.ensure_future, callback(*args, **kwargs))
```

Comment:
Is it a breaking change?
Please have a look at the following PR: https://github.com/hyperledger/indy-plenum/pull/974/files

Reply:
It shouldn't be - according to [this part of Python 3.5 documentation](https://docs.python.org/3.5/library/asyncio-task.html#asyncio.async) `async` is just a wrapper for `ensure_future` and was already deprecated by that time.